solidity/app/contracts/Router.sol

```diff
@@ -30,7 +30,10 @@ abstract contract Router is AbacusConnectionClient, IMessageRecipient {
      * @param _router The address the message is coming from
      */
     modifier onlyRemoteRouter(uint32 _origin, bytes32 _router) {
-        require(_isRemoteRouter(_origin, _router), "!router");
+        require(
+            _isRemoteRouter(_origin, _router),
+            "No router enrolled for domain. Did you specify the right domain ID?"
```

Comment:
can we dedupe these?

Reply:
Doesn't *really* matter but worth mentioning anyways - some consider it a good practice to have revert strings less than 32 bytes, wondering how we feel about that

Reply:
Moved it out as a constant.

I would say at this stage, easy to understand error message >> gas